api/specs/web-server/_projects_crud.py

```diff
@@ -109,3 +112,14 @@ async def clone_project(
     _params: Annotated[ProjectPathParams, Depends()],
 ):
     ...
+
+
+@router.get(
+    "/projects/{project_id}/inactivity",
```

Comment:
I am not sure I consider the inactivity as a resource of a project.
I guess @pcrespov would not like to add this into the ProjectGet object but well...
what is the concept here? is the frontend going to poll this endpoint? or is there an event triggered from the backend?

Reply:
The idea here is the following:
- when the frontend detects that the user was inactive for 30 minutes (default value), it will ask the backend if it can close
- this "inactivity" call is wired to the project. Now I'm also not sure if it should be a resource or not, now that you pointed it out. I guess we can also use a `/projects/{project_id}:inactive` endpoint.

Reply:
@sanderegg but this entrypoint avoids adding it to ProjectGet which is IMO better. This is a property of the referenced project expressed as a https://cloud.google.com/apis/design/design_patterns#singleton_resources

@GitHK what makes you think that `/projects/{project_id}:inactive` is even a possibility? I am a bit worried that at this point are still discussing these things. Please read carefully https://cloud.google.com/apis/design/custom_methods

Reply:
@pcrespov tbh I'm still a bit unsure about when and what can be done in some cases. yes it is not very clear. And the link you provided to the singleton_resources does not really cover the inactivity. It is a read-only status inside the project resource. Maybe we can discuss that later so I understand the concept?

Reply:
@sanderegg you are right. it is read-only (does not need update) and it defines a state of the project. I am wrong. It is not a singleton resource as defined in the link above. My mistake :-)